src/OrchardCore/OrchardCore.DisplayManagement/Handlers/DisplayDriver.cs

```diff
@@ -72,14 +72,19 @@ public virtual Task<bool> CanHandleModelAsync(TModel model)
 
         public virtual Task<IDisplayResult> DisplayAsync(TModel model, TDisplayContext context)
         {
-            return DisplayAsync(model, context.Updater);
+            return Task.FromResult(Display(model, context));
```

Comment:
I'm not sure about this one, since this is a subtly breaking change: Previously, if you've overridden `Task<IDisplayResult> DisplayAsync(TModel model, IUpdateModel updater)` then you also received `Task<IDisplayResult> DisplayAsync(TModel model, TDisplayContext context)` calls. Now you won't. It also goes against the patterns of `IDisplayResult Display(TModel model, TDisplayContext context)`.

Reply:
@Piedone you are right. I think we should replace `Task<IDisplayResult> DisplayAsync(TModel model, IUpdateModel updater)` with `Task<IDisplayResult> DisplayAsync(TModel model, TDisplayContext context)` which is another breaking change to the display driver. This isn't bad since we already had another breaking change earlier from Jasmine in the display driver. Thoguths on replacing the parameter instead?

Reply:
I'm inclined to say yes, because the `IUpdateModel` overrides are quite strange for me too, and there's no usage of it them in [OSOCE](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions) either (just one that shouldn't be as it doesn't use the updater). But I'm surely missing something. @Skrypt can you help, please?

Reply:
@Piedone @Skrypt I made the changes in this PR because this is the only way this PR would work. Also, I think it's less overrides we have to manage. Please review this and let me know your thoughts.

Reply:
Here is the call stack path

### Display 
`DisplayAsync(TModel model, TDisplayContext context)` >> `Display(TModel model, TDisplayContext context)` >> `Display(TModel model)`

### Edit
`EditAsync(TModel model, TEditorContext context)` >> `Edit(TModel model, TEditorContext context)` >> `Edit(TModel model, IUpdateModel updater)` >> `Edit(TModel model)`

### Update
`UpdateAsync(TModel model, TUpdateContext context)` >> `UpdateAsync(TModel model, IUpdateModel updater)` >> `Edit(model, updater)`